src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -650,6 +677,25 @@ public class DeleteDataFilePlan
 
     public class DeleteReleasePlan
     {
-        public List<IdTitleViewModel> ScheduledMethodologies { get; set; } = new();
+        public IReadOnlyList<IdTitleViewModel> ScheduledMethodologies { get; init; } = [];
+
+        public IReadOnlyList<DeleteApiDataSetVersionPlan> ApiDataSetVersions { get; init; } = [];
+
+        public bool Valid => ApiDataSetVersions.All(dsv => dsv.Valid);
+    }
+
+    public class DeleteApiDataSetVersionPlan
+    {
+        public Guid DataSetId { get; init; }
+
+        public string DataSetName { get; init; } = null!;
+
+        public Guid DataSetVersionId { get; init; }
```

Comment:
Could this be simply `Id` given that the view model is concerning a data set version?